vars/piperPipelineStageRelease.groovy

```diff
@@ -21,6 +21,8 @@ import static com.sap.piper.Prerequisites.checkScript
     'tmsUpload',
     /** Publishes release information to GitHub. */
     'githubPublishRelease',
+    /** Executes smoke tests by running the specified npm scripts. */
```

Comment:
Should this instead say that it runs the "ci-smoke" script? With "specified" I would assume that the script is user-configured (I mean not the script itself, but "which one" shall run).

Reply:
Yes you are right makes sense!

Reply:
I'll address it.